libs/components/toast/src/lib/modules/toast/toast.component.scss

```diff
@@ -30,66 +133,94 @@ sky-toast {
   }
 
   .sky-toast-content {
-    padding-top: $sky-padding;
-    padding-bottom: $sky-padding;
+    padding: var(
+      --sky-override-toast-content-padding,
+      0 var(--sky-space-gap-text_action-m) 0 var(--sky-space-gap-icon-l)
+    );
     width: 100%;
     word-wrap: break-word;
     overflow: hidden;
 
     a,
     .sky-btn-link,
     .sky-btn-link-inline {
-      color: change-color($sky-text-color-default, $alpha: 0.8);
-      text-decoration: underline;
+      color: var(
+        --sky-override-toast-content-link-color,
+        var(--sky-color-text-default)
+      );
+      text-decoration: var(
+        --sky-override-toast-content-link-decoration,
+        var(--sky-font-text_decoration-visible_link)
+      );
 
       &:hover {
-        color: $sky-text-color-default;
+        color: var(--sky-text-color-default);
```

Comment:
I think this is one of the old tokens, and it should be `--sky-color-text-default` now